src/components/forms/SegmentedToggle/SegmentedToggleItem.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { includes } from 'ramda';
+import { noop } from 'ramda-adjunct';
+
+import TabLabel from '../../Tabs/TabLabel';
+import { SegmentedToggleSizes } from './SegmentedToggle.enums';
+import { Radio } from '..';
+import { TabsEnums } from '../..';
```

Comment:
Just a note be careful with these kind of imports like `..` or `../..` I'm not sure why VSCode is doing this but several times I hit an issue that build was failing. 
Rather use `../Radio` or `../../Tabs`.